### DISCUSSION TITLE
Does DeePMD-kit apply any constraints to establish the total force as zero?

### DISCUSSION THREAD

**Discussion by mornida** [deepmodeling/deepmd-kit#2699] [Q&A] [answered]
Hello,

I utilized DeePMD-kit version 2.1.5 to create a model and conducted molecular dynamics using LAMMPS within a periodic boundary condition. Upon observation, I noticed that the sum of forces acting on atoms is almost zero in every direction.

I have a question about whether DeePMD-kit applies any constraints to establish the total force as zero. If so, should I create training data where the total force is zero?

Thank you for your help.

<img width="477" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/84767728/27c6e81e-39f3-4f7b-b9a7-234e99df8251">

**Comment by njzjz** [comment, marked as answer]
The Third Newton's law applies.
